pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: genkit
-description: Genkit client library
-version: 0.0.2
+description: A type-safe Dart client library for calling Google Genkit flows with streaming support, authentication, and comprehensive error handling.
+version: 1.0.0
```

Let's keep it "unstable" for now. We have a few ideas in the pipeline for the client SDKs.

```suggestion
version: 0.5.0
```